src/components/PageNotFound/index.js

```diff
@@ -8,7 +8,7 @@ export const PageNotFound = ({ image, mainText, subText }) => {
     <div className="page-not-found-component">
       <div className="inner-container">
         <img src={withPrefix(image)} alt='404'/>
-          <h1>{mainText}</h1>
+          {!image || (<h1>{mainText}</h1>)}
```

Comment:
why is image prop used to null check mainText?

Reply:
My bad, mainText should be used to null check mainText.